designing-your-journal/en/creating-stylesheet.md

```diff
@@ -0,0 +1,69 @@
+# Create a Stylesheet
+
+In OJS, the Journal Manager, Journal Editor and Production Editor can upload a customized CSS (cascading stylesheet) file to override default CSS settings of a theme.
+
```

Comment:
Can we put a more strongly worded caution in here? Something like:

> When customizing your journal with custom CSS styles, we strongly encourage you to involve technical and design experts to ensure that the changes you make do not adversely effect your journal. Changes to the CSS can effect your [compliance with accessibility](./inclusive-and-accessible-theming) laws in your country, impact the usability of your journal on different devices, and cause other unforeseen consequences that a professional can help you avoid.
>
> However, we understand that many journals do not have the funds to access these resources. The guide below is provided for those who want to try to write CSS customizations themselves.

Maybe we can add a `{:.warning}` tag to it to make it stand out? That can be done like this:

```
> This is the warning message.
{:.warning}
```

Reply:
I see that there is a caution in the introduction. Still, it might be worth reiterating the caution at the start of this page.